app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -149,7 +149,7 @@ register_katello_host(){
   params << "update_packages=#{@update_packages}" unless @update_packages.nil?
 -%>
   UUID=$(subscription-manager identity | grep --max-count 1 --only-matching '\([[:xdigit:]]\{8\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{12\}\)')
-  <%= indent(2, skip1: true) { generate_web_request(utility: @download_utility, url: @registration_url, ssl_ca_cert: "$KATELLO_SERVER_CA_CERT", headers: headers, params: params) } %>
+  <%= indent(2, skip1: true) { generate_web_request(utility: @download_utility, url: @registration_url, ssl_ca_cert: "$KATELLO_SERVER_CA_CERT", output_file: "/root/registration_host_init.sh", headers: headers, params: params) } %>
```

Comment:
is the UUID accessible from inside the (new) script? my head says it won't be, but I haven't tested it.